lib/facade_converter.ts

```diff
@@ -209,6 +214,10 @@ export class FacadeConverter extends base.TranspilerBase {
     fileName = fileName.replace(/(\.d)?\.ts$/, '');
     fileName = fileName.replace(FACADE_NODE_MODULES_PREFIX, '');
 
+    if (this.typingsRootRegex != null) {
+      fileName = fileName.replace(this.typingsRootRegex, '');
```

Comment:
can we make another local variable `canonicalFileName` so it's more clear what is happening